common/src/provisioning/Authentication/ServiceConnectionString.cs

```diff
@@ -113,11 +126,20 @@ public string GetPassword()
             return password;
         }
 
+        /// <summary>
+        /// Returns the Authorization header value
+        /// </summary>
+        /// <returns></returns>
         public string GetAuthorizationHeader()
```

Comment:
We should not mix things up - this is a connection string parser that should not get a particular HTTP style header.

Reply:
The token generation logic has been removed from the parser